### ISSUE TITLE
Registration redirect page is empty

### ISSUE THREAD

**Issue by ZerooCool** [joomla/joomla-cms#23937]
### Steps to reproduce the issue
Go to : https://www.visionduweb.fr/component/users/?view=registration
Create a account
Redirect auto to : https://www.visionduweb.fr/component/users/?view=registration&layout=complete 
This page show the confirmation from new registration, need the mail validation.

### Expected result
Redirection to index.php or to a specific article (Please validate your account from your email)

### Actual result
the content page is empty

### System information (as much as possible)
Joomla stable.
PHP 7.2
Template ETConference

### Additional comments
I have no idea if it comes from the template or Joomla but I think it should be able to manage the redirection for the user who has just created his account, but not yet registered.

**Comment by ZerooCool** [comment]
I see that : 
https://transfernow.net/ddl/capture

**Comment by ghost** [comment]
@HLeithner can you please comment?

**Comment by HLeithner** [comment]
file doesn't exists on transfernow.

@ZerooCool Is this a new problem after an upgrade or is this site new? Did you set the redirect in the menu item?

I think its not a Joomla! core problem, more a problem of the template. You should ask in the forum for help.

**Comment by HLeithner** [comment]
Btw. which Joomla version do you use?

**Comment by ghost** [comment]
@ZerooCool Please ask on [Forum](https://forum.joomla.org/) as suggested by @HLeithner.

**Comment by ghost** [comment]
closed as stated above. Issue can always be reopened.

**Comment by ZerooCool** [comment]
From what I've been looking for, this behavior has been normal for a long time on Joomla.

After registering a user, we arrive on a page that displays the message "Confirm your user account, from the mail".

The body of this page is empty.
Only the account creation confirmation message is displayed.

I do not know if some people know better than others, not to have a blank page, but, I found a message about Joomla 2.5 with a user who already had this type of display (empty page. )

I tested on my site that comes from an updated version 2.5 with a Vertex template,
I tested on my new site which is a version 3.9 updated and stable with another template than Vertex, I have the same behavior.

User registration, ok.
Message confirmation is displayed, ok.
The body of this page contains only the confirmation message.

It's a little bare as a type of display.
It does not sound like a bug, but, a desired behavior.
It remains very sober, too sober.

This could be improved.
